README.md

```diff
@@ -93,7 +93,7 @@ The community holds office hours every week, with sessions open to all users and
 developers.
 
 Office hours are hosted on a zoom video chat every Thursday
-at 09:00 (PT) / 12:00 (ET) / 17:00 (UTC),
+at 09:00 (PT) / 12:00 (ET) / 16:00 (UTC),
```

Comment:
> If you specify a time (rather than a time zone), you should use the correct one (PDT or PST).

So to be accurate, should we merge this change, then revert it on November 6 when Daylight Savings ends?